mappings/net/minecraft/util/ChatCharacterFilter.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS k
+CLASS k net/minecraft/util/ChatCharacterFilter
```

Comment:
Judging from realm's `RealmsSharedConstants`, this is more likely `net/minecraft/SharedConstants`. It'll be the obfuscator removing fields that are inlined, as to why it doesn't fell much like a typically constants class.